osu.Game.Tests/Visual/Online/TestSceneUserPanel.cs

```diff
@@ -51,27 +62,40 @@ public class TestSceneUserPanel : OsuTestScene
                         IsSupporter = true,
                         SupportLevel = 3,
                     }) { Width = 300 },
+                    evast = new UserListPanel(new User
+                    {
+                        Username = @"Evast",
+                        Id = 8195163,
+                        Country = new Country { FlagName = @"BY" },
+                        CoverUrl = @"https://assets.ppy.sh/user-profile-covers/8195163/4a8e2ad5a02a2642b631438cfa6c6bd7e2f9db289be881cb27df18331f64144c.jpeg",
+                        IsOnline = false,
+                        LastVisit = DateTimeOffset.Now
+                    })
                 },
             };
 
             flyte.Status.Value = new UserStatusOnline();
+
             peppy.Status.Value = null;
             peppy.Activity.BindTo(activity);
+
+            evast.Status.Value = null;
+            evast.Activity.BindTo(activity);
         });
 
         [Test]
         public void TestUserStatus()
         {
-            AddStep("online", () => peppy.Status.Value = new UserStatusOnline());
-            AddStep("do not disturb", () => peppy.Status.Value = new UserStatusDoNotDisturb());
-            AddStep("offline", () => peppy.Status.Value = new UserStatusOffline());
-            AddStep("null status", () => peppy.Status.Value = null);
+            AddStep("online", () => peppy.Status.Value = evast.Status.Value = new UserStatusOnline());
+            AddStep("do not disturb", () => peppy.Status.Value = evast.Status.Value = new UserStatusDoNotDisturb());
+            AddStep("offline", () => peppy.Status.Value = evast.Status.Value = new UserStatusOffline());
+            AddStep("null status", () => peppy.Status.Value = evast.Status.Value = null);
```

Comment:
You can extract one local bindable and do `BindTo()` twice (the same way as it's done with `Activity`) to avoid compound assignment.